### DISCUSSION TITLE
Running headless/without a gui

### DISCUSSION THREAD

**Discussion by Bindernews** [bevyengine/bevy#4860] [Q&A] [answered]
I'm planning to build a server component for a game so it'll need to be able to run headless. Does Bevy support this?

For context, I'm trying to build a simulator for a specific part of the game. And no, I unfortunately cannot re-use the original game. It's not designed for that in any way.

**Comment by bjorn3** [comment, marked as answer]
I believe you can do something like https://github.com/bevyengine/bevy/blob/c89af06c6558b80e1ff61e945124625944356cd2/examples/app/headless_defaults.rs#L8-L12 since https://github.com/bevyengine/bevy/pull/3439.

**Reply by Bindernews** [reply]
I did look in the examples, but I didn't see that because it was inside the "app/" folder. Guess I should have searched the repo code and not just the discussions. Thanks!
